File: Lesson_1/FrogJmp/C++/solution.cpp
// you can use includes, for example:
// #include <algorithm>

// you can write to stdout for debugging purposes, e.g.
// cout << "this is a debug message" << endl;
#include <math.h> 

int solution(int X, int Y, int D) {
    // write your code in C++11
    unsigned int distance = Y-X;
    int result = ceil(double(distance)/D);
    return result;
}

File: Lesson_1/FrogJmp/Javascript/solution.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(X, Y, D) {
    // write your code in JavaScript (Node.js 4.0.0)
    var distance = Y - X;
    var result = Math.ceil(distance/D);
    return result;
}